Rename all methods IsEmpty to Empty to be consistent with all codebase (#6409)
cosmos/cosmos-sdk: CHANGELOG.md

### API Breaking Changes

> * (code hygiene) [\#6409](https://github.com/cosmos/cosmos-sdk/pull/6409) Rename all IsEmpty methods to Empty in all codebase for consistency.

Reviewer: ```suggestion
* (code hygiene) [\#6409](https://github.com/cosmos/cosmos-sdk/pull/6409) Rename all IsEmpty methods to Empty across the codebase and enforce consistency.
```

### Improvements

> * (code hygiene) [\#6409](https://github.com/cosmos/cosmos-sdk/pull/6409) Rename all IsEmpty methods to Empty in all codebase for consistency.

Reviewer 2: This is technically an API breaking change.
